include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1928,6 +1934,11 @@ class MatrixFree : public Subscriptor
    */
   mutable std::list<std::pair<bool, AlignedVector<Number>>>
     scratch_pad_non_threadsafe;
+
+  /**
+   * Stored the level of the mesh to be worked on.
+   */
+  unsigned int level_mg_handler;
```

Comment:
And the same is true here -- maybe `mg_dof_handler_level` or `mg_level` would have been easier to understand?

Reply:
I like `mg_level` as the variable name and `get_mg_level()` for the access function; it is good to imply that the `level` relates to multigrid functionality and the prefix `mg` in line with most of our other names elsewhere, like `DoFHandler::distribute_mg_dofs()`, `DoHandler::locally_owned_mg_dofs()`. On the other hand, we use the `level` word in a few spots as well, like `DoFTools::extract_level_dofs()` or `DoFTools::extract_locally_relevant_level_dofs()` (rather than `..._mg_dofs`), so in that sense this function would contain both parts and use a word that we have not used in the library before (I grepped for `mg_level` in `include` and `source` and only found the member variable `mg_levels` and some header file names). The name is better than `get_level()` alone given its ambiguity, and better than `get_multigrid_level()`, as we hardly spell out the `multigrid` word for level information.